exception/GraknClientException.java

```diff
@@ -45,7 +45,10 @@ public static GraknClientException create(String error) {
         return new GraknClientException(error);
     }
 
-    public static GraknClientException create(String error, StatusRuntimeException e) {
+    public static GraknClientException create(String error, Exception e) {
+        if (e instanceof StatusRuntimeException) {
+            return new GraknClientException(error, (StatusRuntimeException) e);
+        }
```

Comment:
What if we just overload the method by like the following, rather than manually downcasting?
```
public static GraknClientException create(String error, StatusRuntimeException e) {
        return new GraknClientException(error, e);
    }
```

Reply:
Can do

Reply:
Actually, no, I remember why I didn't want to do that, it's because the `Exception` might be a `StatusRuntimeException` but the compile time type is an `Exception` (caught as `Exception` for a more general case).

Reply:
Since the logic to extract error code is different, it's important it gets treated as a special case even if it was upcast to an `Exception` before the method call.

Reply:
Right. I suspected that. And I wasn't sure which method would be called by java. 🤔

In any case, your way is safer.